sips/mev.md

```diff
@@ -0,0 +1,111 @@
+
+The SSV contract must allow validators to update this attachment, as their preferences may change over time.
+
+Operators must validate the signature and keep track of their validators' latest attachment (by highest `timestamp`), and periodically publish it to their Beacon nodes at [/eth/v1/validator/register_validator](https://ethereum.github.io/beacon-APIs/#/Validator/registerValidator).
+
+An operator may publish for his entire validator set in bulk, however a single invalid signature may cause the request to fail and other attachments to be ignored, so a prior signature validation is crucial.
+
+> Note: producing a `BlindedBeaconBlock` before and shortly thereafter (delay depends on the builder) the registration may result in either:
+> 1. Beacon node falls back to locally-built block from it's execution layer.
+> 2. Builder doesn't reward the validator's `fee_recipient` because it isn't aware of it.
+> 3. Builder rewards a potentially different `fee_recipient` from the validator's latest registration (such as a registration prior to onboarding to SSV.)
+
+#### Issues
+
+##### Gas limit
+
+Within the proposed construction, the gas limit is set by validators only and relies on them to update it if needed.
+
+If gas limit is likely to change to adjust to network conditions, then this isn't ideal because validators would have to track of it, and when it changes — produce a new signature and publish an update transaction to the contract (which may be very expensive when the network is congested).
+
+> TODO: examine the code of validator clients such as Lighthouse and Prysm to figure this out.
+
+### Blinded block proposals
+
+#### Scenario A — Block is published only by operators who share builder with leader
+
+1. Leader produces a `BlindedBeaconBlock` from their Beacon node at [/eth/v1/validator/blinded_blocks](https://ethereum.github.io/beacon-APIs/#/Validator/produceBlindedBlock) and broadcasts it.
+
+2. Operators sign the `BlindedBeaconBlock`.
+
+3. Operators publish the `SignedBlindedBeaconBlock` to their Beacon node at [/eth/v1/beacon/blinded_blocks](https://ethereum.github.io/beacon-APIs/#/Beacon/publishBlindedBlock).
+  
+    The Beacon node reveals the signature to the builder, which (ideally) reveals the transactions in return, after which it unblinds the block and publishes it to the network.
+
+    ##### Issues
+
+    Unrealistic?
+
+#### Secnario B — Block is published by all operators
+
+1. Leader produces a `BlindedBeaconBlock` from their Beacon node at [/eth/v1/validator/blinded_blocks](https://ethereum.github.io/beacon-APIs/#/Validator/produceBlindedBlock) and broadcasts it together with the endpoint of it's builder as `BuilderBlindedBeaconBlock`.
+
+    ```go
+    type BuilderBlindedBeaconBlock {
+        BuilderEndpoint    string
```

Comment:
Is it the builder or relay endpoint ?

Reply:
also seems super easily abused

Reply:
> Is it the builder or relay endpoint ?

I call it "builder" instead of "relay" because both relays and individual builders implement the [builder-specs](https://github.com/ethereum/builder-specs), so theoretically you should be able to point your Beacon node to an individual builder (if you have some privileged access to them?) instead of a relay.

> also seems super easily abused

Yeah, operators trust the leader on this one :\